SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Modules/SalemoveSDK.swiftmodule/arm64-apple-ios.swiftinterface

```diff
@@ -543,6 +546,17 @@ extension SalemoveSDK.Salemove {
   @objc final public func requestEngagedOperator(completion: @escaping SalemoveSDK.OperatorBlock)
   @objc final public func endEngagement(completion: @escaping SalemoveSDK.SuccessBlock)
 }
+public enum EngagementSource : Swift.RawRepresentable, Swift.Codable {
+  case coreEngagement
+  case callVisualizer
+  case unknown(Swift.String)
```

Comment:
How do you start with unknown?

Reply:
This structure is used for another purpose and the unknown case is expected here. For the clients of Glia, additional information about integration can be found in our integration guide and questions about internal implementation details can be discussed by contacting customer support.

Reply:
This structure is used for another purpose and the unknown case is expected here.
For clients of Glia, additional information about integration can be found in our integration guide and questions about internal implementation details can be discussed by contacting customer support.